src/Components/Aspire.Oracle.EntityFrameworkCore/OracleEntityFrameworkCoreSettings.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Oracle.EntityFrameworkCore;
+
+/// <summary>
+/// Provides the client configuration settings for connecting to a Oracle database using EntityFrameworkCore.
+/// </summary>
+public sealed class OracleEntityFrameworkCoreSettings
+{
+    /// <summary>
+    /// The connection string of the Oracle database to connect to.
+    /// </summary>
+    public string? ConnectionString { get; set; }
+
+    /// <summary>
+    /// Gets or sets a boolean value that indicates whether the db context will be pooled or explicitly created every time it's requested.
+    /// </summary>
+    public bool DbContextPooling { get; set; } = true;
+
+    /// <summary>
+    /// <para>Gets or sets the maximum number of retry attempts.</para>
+    /// <value>
+    /// The default is 6.
+    /// Set it to 0 to disable the retry mechanism.
+    /// </value>
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// <para>Gets or sets the maximum number of retry attempts.</para>
    /// <para>Default value is 6, set it to 0 to disable the retry mechanism.</para>
    /// </summary>
```

`value` isn't a valid element in summary. Repeated elsewhere. See

https://github.com/dotnet/aspire/blob/bc220e3de6d8b398b22e0d0b6d338a8ab1ab1ba1/src/Components/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL/NpgsqlEntityFrameworkCorePostgreSQLSettings.cs#L25-L37

for the latest format.

Reply:
All right